staff/jaime/taskApi-Client_Server_Alex&Jaime/tasks-api-client/src/taskApi.js

```diff
@@ -0,0 +1,32 @@
+const axios = require('axios')
+
+let taskApi;
+
+(function () {
+
+    const baseUrl = 'http://localhost:5000/api'
+
+    taskApi = {
+
+        //Peticion de todas las tareas
+        getTaskList: function () {
+            return axios.get(`${baseUrl}/show/tasks`)
+                .then(res => res.data)
+        },
+
+        setTask: function (task) {
```

Comment:
I would better use createTask instead of setTask, as you have it in the url /create/task